build/ToolsetPackages/BaseToolset.csproj

```diff
@@ -27,5 +27,6 @@
     <PackageReference Include="RoslynTools.ReferenceAssemblies" Version="$(RoslynToolsReferenceAssembliesVersion)" />
     <PackageReference Include="GitLink" Version="$(GitLinkVersion)" />
     <PackageReference Include="Microsoft.VSSDK.BuildTools" Version="$(MicrosoftVSSDKBuildToolsVersion)" />
+    <PackageReference Include="xunit.runner.wpf" Version="$(xunitrunnerwpfVersion)" />
```

Comment:
Ah! I guess this happens here now.

Reply:
Yeah it had to move here. In the other position it was adding items to the reference graph and the SDK was unhappy. Here this is a simple net461 project for restoring tools so it fits more naturally.

Reply:
Makes sense